client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -109,6 +109,9 @@ public final class ApiEndpointsV1
     public static final String ABORT_RESTORE_JOB_ROUTE = RESTORE_JOB_ROUTE + ABORT;
     public static final String RESTORE_JOB_PROGRESS_ROUTE = RESTORE_JOB_ROUTE + PROGRESS;
 
+    public static final String LIST_JOBS_ROUTE = API_V1 + CASSANDRA + "/jobs";
+    public static final String JOB_STATUS_ROUTE = API_V1 + CASSANDRA + "/jobs/" + JOB_ID_PATH_PARAM + "/status";
```

Comment:
`/status` feels redundant. 

`.../jobs/:jobId` can be used to retrieve the job details, including status. The job details API is not there. Is it reserved for later?

Reply:
> The job details API is not there. Is it reserved for later?

This is the only job API since it would return the job status, operation and failure reason when applicable. I do not see a use-case for a separate details API, although I do agree we could do away with the `status` suffix.

Reply:
Yep. I mean exactly this, replacing `jobs/:jobId/status` with `jobs/:jobId`